alcs-frontend/src/app/features/notice-of-intent/notice-of-intent.component.ts

```diff
@@ -90,6 +92,10 @@ export class NoticeOfIntentComponent implements OnInit, OnDestroy {
         this.noticeOfIntent = noticeOfIntent;
       }
     });
+
+    this.noticeOfIntentModificationService.$modifications.pipe(takeUntil(this.destroy)).subscribe((value) => {
+      this.modifications = value;
```

Comment:
Do you need to clear this.modifications in OnDestroy?

Reply:
Ohh good catch, definitely should